CONTRIBUTING.md

```diff
@@ -40,5 +40,5 @@ mix deps.get
 ## Running tests
 
 ```bash
-mix test
+mix coveralls.json
```

Comment:
Do we want this change? Normally, I'd only run coveralls in CI, but I'd use `mix test` locally because it's faster and I don't need the coverage regenerated on every test run when I'm doing local development.

#optional

Reply:
Thanks Randy, I'll put test back.